client/src/components/body/lab5/reading.js

```diff
@@ -0,0 +1,132 @@
+import React, {useEffect} from "react";
+import {LAB_ID} from '../../../constants/lab5';
+import UserLabService from '../../../services/UserLabService';
+import { Pie } from "react-chartjs-2";
+
+const data = {
+    labels: [
+        "US Population (Millions)",
+        "Cognitively Impaired (Millions)"
+    ],
+    datasets: [
+        {
+            label: "Cognitively Impaired in a Population of 328 People",
+            borderColor: "black",
+            backgroundColor: ["blue", "red"],
+            data: [328, 16],
+            borderWidth: "1"
+        }
+    ]
+};
+
+const Reading = () => {
+    useEffect(() => {
+        return () => {
+            UserLabService.complete_reading(LAB_ID);
+        }
+    });
+    return (
+        <div className="study">
+            <h3>What is Cognitive Impairment?</h3>
+            <p>
+                Cognitive impairment refers to a broad range of disabilities, from people with intellectual disabilities who may have the most-limited capabilities, to age-related issues with thinking and remembering. The range includes people with learning disabilities, such as dyslexia and attention deficit hyperactivity disorder (ADHD)
+            </p>
+            <h3>How many people are affected?</h3>
```

Comment:
This is a little thing, but it looks like subheads are capitalized in other labs so it might be good to stay consistent and make this capitalized ("How Many People Are Affected?")